bin/env.sh.template

```diff
@@ -3,17 +3,17 @@ export CLASS_PATH="target/lib/*"
 export ZK_QUORUM=zk.host:2181
```

Comment:
There shouldn't be a need to change anything in this file

Reply:
Sure